### PR TITLE
Added FPS label for the UI

### PR DESCRIPTION
**Pull request by avrahamshukron**
Signed-off-by: Avi Shukron <avraham.shukron@gmail.com>

### CODE REVIEW

File: graphics/alert_bar.py [hunk 1]
```diff
@@ -30,6 +31,11 @@ def __init__(self, parent, store):
         self.bar = Frame(self.root, bg='green', height=self.height, width=self.width)
         self.message_label = Label(master=self.bar, font=("Courrier", 40),
                                    text="OK", bg='green', fg='black')
+        self.fps_label = Label(
+            master=self.bar, font=("Arial", 20),
+            text="FPS: 0", bg='green', fg='black')
+        self.fps_label.place(anchor="ne", relx=0.95, rely=0.3)
+        self.last_update = time.time()
```
Comment (rinslow) [inline graphics/alert_bar.py]: self.last_n_updates = [time.time()] * 5

**Comment by avrahamshukron** [comment]
I want to postpone this PR. It is mostly a debug feature and maybe we don't want it for production

**Comment by rinslow** [comment]
> I want to postpone this PR. It is mostly a debug feature and maybe we don't want it for production

This should be merged after https://github.com/Reznic/Inhalator/pull/14 then, and set to work in simulation mode

**Comment by avrahamshukron** [comment]
> > I want to postpone this PR. It is mostly a debug feature and maybe we don't want it for production
> 
> This should be merged after #14 then, and set to work in simulation mode
great idea

**Comment by avrahamshukron** [comment]
We  are way past that now...

**Pull request closed without merging**